src/styles.css

```diff
@@ -71,8 +71,9 @@ div.Node div.Status,
 div.Node div.Cors,
 div.Node div.Origin,
 div.Node div.Trustless,
+div.Node div.Ipns,
 div.Node div.Flag {
-	width: 6em;
+	width: 7.2em;
```

Comment:
weird way these divs are structured. I'll create an issue to have a better table replacement here.

Reply:
https://github.com/ipfs/public-gateway-checker/issues/314